### PR TITLE
Update dependencies and maintainers

### PR DESCRIPTION
**Pull request by bmagyar**
To help these packages take off I'd like to do a 0.0.1 release to Rolling very soon.

These changes prepare things for that but we should also have the tests fixed as per #3

### CODE REVIEW

**Review by pac48 — APPROVED** [review]
LGTM

File: kinematics_interface_kdl/package.xml [hunk 1]
```diff
@@ -4,15 +4,23 @@
...
+  <build_depend>eigen</build_depend>
 
   <depend>kinematics_interface</depend>
+  <depend>pluginlib</depend>
+  <depend>kdl_parser</depend>
 
-  <test_depend>ament_lint_auto</test_depend>
-  <test_depend>ament_lint_common</test_depend>
+  <test_depend>ros2_control_test_assets</test_depend>
+  <test_depend>ament_cmake_gmock</test_depend>
```
Comment (AndyZe) [inline kinematics_interface_kdl/package.xml:23]: nitpick: may want to alphabetize.

File: kinematics_interface_kdl/CMakeLists.txt [hunk 2]
```diff
@@ -14,14 +14,14 @@ find_package(kinematics_interface REQUIRED)
...
-        )
+)
 
 ament_target_dependencies(kinematics_interface_kdl
         kdl_parser
         pluginlib
         Eigen3
         kinematics_interface
-        )
+)
```
Comment (AndyZe) [inline kinematics_interface_kdl/CMakeLists.txt:24]: L61 needs an indentation fix, too

File: kinematics_interface/CMakeLists.txt [hunk 3]
```diff
@@ -17,8 +17,7 @@ find_package(rclcpp_lifecycle REQUIRED)
 # Create interface library for kinematics base class
 add_library(kinematics_interface
     INTERFACE
-    include/kinematics_interface/kinematics_interface_base.hpp
-    )
+)
```
Comment (AndyZe) [inline kinematics_interface/CMakeLists.txt:20]: L40 needs an indentation fix too

File: kinematics_interface_kdl/CMakeLists.txt (hunk 2 above)
Comment (bmagyar) [reply kinematics_interface_kdl/CMakeLists.txt:24]: done

File: kinematics_interface/CMakeLists.txt (hunk 3 above)
Comment (bmagyar) [reply kinematics_interface/CMakeLists.txt:20]: done

File: kinematics_interface_kdl/package.xml (hunk 1 above)
Comment (bmagyar) [reply kinematics_interface_kdl/package.xml]: done

**Review by AndyZe — APPROVED** [review]

**Pull request merged**